aggregator/src/aggregator/collection_job_driver.rs

```diff
@@ -386,13 +388,43 @@ where
         )
         .await?;
 
+        let response_headers = http_response.headers();
+        let response_body = http_response.body();
+        if response_body.is_empty() {
+            let retry_delay = match parse_retry_after_to_duration(response_headers) {
+                Ok(Some(retry_delay)) => retry_delay,
+                _ => self
+                    .collection_retry_strategy
+                    .clone()
```

Comment:
I'm surprised you need to `clone()` here since `RetryStrategy::compute_retry_delay` takes `&self`
https://github.com/divviup/janus/blob/6205834e3d4b281f506781b544b3f242aca815e2/aggregator/src/aggregator/collection_job_driver.rs#L790